custom/icds_reports/utils/aggregation_helpers/distributed/bihar_api_demographics.py

```diff
@@ -134,6 +149,13 @@ def aggregation_query(self):
               );
                 """
 
+    def indexes(self):
```

Comment:
This index creation was removed in the other PR which is merged in master. So you can remove this.